src/components/UserInfo/UserInfo.jsx

```diff
@@ -1,3 +1,7 @@
-export const UserInfo = () => (
-  <>Put the user here</>
+import './UserInfo.scss';
+
+export const UserInfo = ({ user }) => (
```

Comment:
Destructure `user`

Reply:
not fixed
<img width="284" alt="image" src="https://github.com/mate-academy/react_static-list-of-posts-js/assets/98764641/1639686f-8e97-40bc-bb44-2be65d63605f">